### DISCUSSION TITLE
Drawmorganbits to smiles

### DISCUSSION THREAD

**Discussion by PARODBE** [rdkit/rdkit#5315] [General] [answered]
Hi,

Any option to convert morganbits to smiles?

Thanks

**Comment by greglandrum** [comment, marked as answer]
@PARODBE apologies for the slow reponse
Does the example in the RDKit documentation do what you want?
https://www.rdkit.org/docs/GettingStartedInPython.html#explaining-bits-from-morgan-fingerprints

**Reply by PARODBE** [reply]
Yes! Thanks @greglandrum !
